terraform/modules/baseline_presets/iam_policy_statements_ec2.tf

```diff
@@ -240,18 +240,7 @@ locals {
       }
     ]
 
-    OracleEnterpriseManager = [
-      {
-        sid    = "S3ListLocation"
-        effect = "Allow"
-        actions = [
-          "s3:ListAllMyBuckets",
-          "s3:GetBucketLocation",
-        ]
-        resources = [
-          "arn:aws:s3:::*"
-        ]
-      },
+    DBSecrets = [
```

Comment:
Is this being used now - I couldn't see it referenced. But if it is, I'd think about changing the name